src/Symfony/Component/Validator/Constraints/CardSchemeValidator.php

```diff
@@ -78,6 +78,10 @@ class CardSchemeValidator extends ConstraintValidator
             '/^5[1-5][0-9]{14}$/',
             '/^2(22[1-9][0-9]{12}|2[3-9][0-9]{13}|[3-6][0-9]{14}|7[0-1][0-9]{13}|720[0-9]{12})$/',
         ),
+        // All UATP card numbers start with a 1 and have a length of 15 digits.
+        'UATP' => array(
+            '/^1([0-9]{14})$/',
```

Comment:
the capturing group is not necessary here.

Reply:
right! changed!